E1_coldflow/brain_utils.h

```diff
@@ -216,21 +265,25 @@ String make_packet(int id, bool error) {
 
 /*
  * Decodes a packet sent from ground station in the following format:
- * {<valve_ID>,<open(1) or close(0)|checksum}
+ * {<ID>,<command (optional)>|checksum}
  * Populated the fields of the valve and returns the action to be taken
  */
-int decode_received_packet(String packet, valveInfo *valve) {
+int decode_received_packet(String packet, valveInfo *valve, sensorInfo *sensor) {
   Serial.println(packet);
```

Comment:
And does this move to the commands library? OR are you planning on keeping it on Common Fw?